tools/schema/cardtree-directory-schema.json

```diff
@@ -6,11 +6,18 @@
   "properties": {
     "directories": {
       "type": "object",
+      "additionalProperties": false,
```

Comment:
Shouldn't we allow any properties at the root level?

Reply:
I think we could. I'll change it.